src/renderers/native/ReactNativeAttributePayload.js

```diff
@@ -314,6 +319,13 @@ function diffProperties(
   for (var propKey in nextProps) {
     attributeConfig = validAttributes[propKey];
     if (!attributeConfig) {
+      if (__DEV__) {
+        var unvalidatedAttributes = ['children', 'collapsable', 'style'];
```

Comment:
I had to opt-out of validation for 'collapsable' and 'style'  to get ReactComponentTreeDevtool-test.native.js to pass. I guess these props come from ReactNativeDefaultInjection's EmptyComponent.